web/admin/src/App.js

```diff
@@ -85,6 +89,14 @@ export default class App extends PureComponent {
     if (lastPublishedAt === undefined) return <div>Loading...</div>
     return (
       <div className="app">
+        <CustomModal
```

Comment:
Does the App component need to know about the modal?  Can it all be inside CurrentContent instead, since that is where the buttons for publishing are?  Then, CurrentContent could call publish only if the user confirms the modal.

Reply:
I believe so because its positioning is contingent on the app box.